databricks_sql_connector_core/src/databricks/sql/auth/retry.py

```diff
@@ -77,7 +77,7 @@ class DatabricksRetryPolicy(Retry):
         commands like ExecuteStatement. This is passed to urllib3 by extending its status_forcelist
 
     :param urllib3_kwargs:
-        Dictionary of arguments that are passed to Retry.__init__. Any setting of Retry() that
+        Dictionary of arguments that are passed to Retry.__init__.py. Any setting of Retry() that
```

Comment:
These `.py` suffixes (?) should not be here - `Retry.__init__` is a magic method, not a file name